src/EditorFeatures/Core/InlineRename/AbstractEditorInlineRenameService.SymbolRenameInfo.cs

```diff
@@ -30,73 +30,47 @@ private partial class SymbolInlineRenameInfo : IInlineRenameInfo
         {
             private const string AttributeSuffix = "Attribute";
 
-            private readonly Document _document;
+            private readonly SymbolicRenameInfo _info;
+
+            private Document Document => _info.Document!;
```

Comment:
`!` is safe as we assert `_info` is in a valid state in the constructor.  Wish there was a nicer way to encode this into NRT.